WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -35,6 +35,13 @@ public class KeyManager
 	public const int MaxGapLimit = 10_000;
 	public static readonly Money DefaultPlebStopThreshold = Money.Coins(0.01m);
 
+	public const byte DefaultShamirShares = 3;
+	public const byte DefaultShamirThreshold = 2;
+	public const int MinShamirShares = 1;
+	public const int MaxShamirShares = 16;
+	public const int MinShamirThreshold = 1;
+	public const int MaxShamirThreshold = 16;
+
```

Comment:
Some of these constants are never used. But they don't belong to the KeyManager class.

Reply:
I think my IDE is laying to me and they are used somewhere.

Reply:
The are all used, now moved to WalletGenerator btw

Reply:
https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/2371fbc0569c05145fb5c3f0e47a0cf2d10fb33c